src/posit/connect/users.py

```diff
@@ -17,6 +18,8 @@ class User(Resource):
 
     Attributes
     ----------
+    content: Content
+        Content resource for user. Sets the content item owner identifier (`owner_guid`) to `self.guid`.
```

Comment:
So this lets you interact with content that is owned by the user?

Reply:
Seemed almost too obvious to ask, but I'm asking to distinguish, does it include stuff they collaborate on?

Reply:
Eh, this isn't a good description. Is this better?

```suggestion
        A content resource scoped to this user.
```


Check this out for context: https://github.com/posit-dev/posit-sdk-py/blob/58a8b9f6ced7cd0c3fd8c8259692fcf33b4b900b/src/posit/connect/content.py#L633-L646

The requests to GET v1/content are scoped with the query param `owner_guid={the guid of this user}`. If my understanding is correct, this will only include content they "own". Not those that they collaborate on.